pkg/syncer/resize_reconciler.go

```diff
@@ -87,7 +87,7 @@ func newResizeReconciler(
 	supervisorNamespace string,
 	resyncPeriod time.Duration,
 	informerFactory informers.SharedInformerFactory,
-	pvcRateLimitter workqueue.RateLimiter,
+	pvcRateLimitter workqueue.TypedRateLimiter[any],
```

Comment:
Is this a required change?

Reply:
workqueue.RateLimiter is deprecated, faced issues in "make check" so needed to fix